ext/mruby_engine/ext.c

```diff
@@ -273,6 +276,23 @@ static VALUE ext_mruby_engine_stat(VALUE rself) {
   uint64_t memory_count = me_mruby_engine_get_memory_count(self);
   rb_hash_aset(stat, ID2SYM(me_ext_id_memory), ULONG2NUM(memory_count));
 
+  int64_t ctx_switches_v = me_mruby_engine_get_ctx_switches_voluntary(self);
+  if (ctx_switches_v >= 0) {
+    rb_hash_aset(stat, ID2SYM(me_ext_id_ctx_switch_v), LONG2NUM(ctx_switches_v));
+  } else {
+    rb_hash_aset(stat, ID2SYM(me_ext_id_ctx_switch_v), Qnil);
```

Comment:
Well, not really given the other changes in https://github.com/Shopify/shopify where we log all keys in the hash (see script_generic_log_usage_quota branch)